src-java/testing/performance-tests/src/test/groovy/org/openkilda/performancetests/BaseSpecification.groovy

```diff
@@ -86,13 +80,21 @@ class BaseSpecification extends Specification {
     @Value('${perf.debug}')
     boolean debug
 
+    //CleanupManager is not called during perf-tests execution due to the implementation
+    //to use automatic cleanup the appropriate listeners should be added
+    static ThreadLocal<CleanupManager> threadLocalCleanupManager = new ThreadLocal<>()
```

Comment:
Listeners(AbstractSpringListener, AutomaticCleanupListener, RegisterListenersExtension) have not been added to avoid auto-cleanup evocation. It was done intentionally (some tests purge topology in the scope of cleanup).  It helps us to avoid unnecessary API calls(delete flow+check history).